.rubocop.yml

```diff
@@ -7,6 +7,10 @@ inherit_mode:
   merge:
     - Exclude
 
+AllCops:
+  Exclude:
+    - db/migrate/201*.rb
```

Comment:
You seem to have fixed issues in lots of these, is this still needed?

Reply:
Yes, unfortunately there are a number of issues that can't be fixed automatically.